### ISSUE TITLE
[Bug]: Luminosity ratio of the focus indicator on the column header of the table is less than required ratio in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#10411]
### Link to reproducible scenario

https://www.ag-grid.com/javascript-data-grid/filtering/

### Describe the bug

Steps to reproduce 

1. Open URL: [JavaScript Grid: Column Filters | AG Grid] (https://www.ag-grid.com/javascript-data-grid/filtering/) on latest Edge browser. 
2. 'Column filters' page will appear.
3. Press tab key to reach to the table present under example.
4. Press tab key to reach to the column header of the table.
5. Turn on Accessibility insights for Windows-color contrast tool.
6. Verify whether luminosity ratio of the focus indicator on the column header of the table is less than required ratio or not.

Actual behaviour
Luminosity ratio of the focus indicator on the column header of the table is 1.8:1 is less than required ratio in Windows

https://github.com/user-attachments/assets/716dba5f-1a6c-4137-9d76-e092e59d816c

<img width="968" alt="Image" src="https://github.com/user-attachments/assets/c1139231-6cb7-4326-9e15-d78f5cd6a275" />

Expected behaviour
Luminosity ratio of the focus indicator on the column header of the table should be greater than or equal to the 3:1 ratio in Windows.

More information
Title: Luminosity ratio of the focus indicator on the column header of the table is less than required ratio in Windows.
User Impact: Low vision users will not be able to identify the currently focused element if luminosity ratio for text is not meeting the minimum required ratio of 3:1.

### Version

30.1.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by ava-utting** [comment]
Hi @msftedad ,

Thank you for raising this with us.

This is an area we aim to improve upon as part of our efforts to enhance accessibility and ensure compliance with WCAG 2.0 standards.
We appreciate you bringing this to our attention.

Kind Regards,
Ava

**Comment by msftedad** [comment]
Hi @ava-utting, Could you please share the reason for closing this issue?

**Comment by ava-utting** [comment]
Hi @msftedad ,

Apologies, I forgot to mention this test case is already covered under the item AG-6163. 

This falls under a broader set of improvements related to theming and accessibility, so it hasn't been logged as a standalone issue.
We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. 

Kind Regards,
Ava
